vars/dockerExecute.groovy

```diff
@@ -142,6 +146,10 @@ void call(Map parameters = [:], body) {
                         body()
                     }
                 } else {
+                    if(!config.dockerName){
+                        config.dockerName = UUID.randomUUID().toString()
```

Comment:
Please add this as a default to the `ConfigurationHelper` above (`addIfEmpty`).

Reply:
But it is only relevant for the case of having sidecar containers.

Reply:
Please keep it there since otherwise it will change the `dockerExdecuteOnKubernetes` behavior!